src/edu/ucsb/cs56/projects/games/fish_animation/FishAnimationEnvironment.java

```diff
@@ -110,8 +110,9 @@ private void gameFinished(boolean won) {
 		Character c = character_type ? Character.SHARK : Character.WHALE;
 		
 		if (won) {
-			message = "Please enter your name \n";
-			playerName = JOptionPane.showInputDialog(fishPanel, message, "CS56");
+			message = "Congratulations! \n You have reached 50 points and win the game!\n Please enter your name \n";
```

Comment:
A couple proofreading nitpicks:
 - Get rid of the spaces next to the newlines - they mess up alignment.
 - It should be "won" not "win".
 - "Please enter your name" should end with a period or a colon.
 - The last newline isn't necessary.